src/components/Controls.svelte

```diff
@@ -1,9 +1,5 @@
 <script lang="ts">
-	import ControlStore from '../stores/ControlStore';
 	import { onDestroy } from 'svelte';
-
-	// context vs store;
-	// can only play one song at a time; player controls was extracted
 	export let songid: string; // required to make modifications to songid (ie add to playlist and more)
```

Comment:
review this

Reply:
Are you suggesting using context over store here?

Reply:
Whats the GitHub Actions lint-and-format complaining about?